From b9c18fa3d55b7d0beb8bc34e51b3b48bf8f985c7 Mon Sep 17 00:00:00 2001
From: Ke Qinghua <qinghua.ke@freescale.com>
Date: Wed, 23 Apr 2014 15:42:08 +0800
Subject: [PATCH 307/602] ENGR00309845 External SD not work both hot plugin
 and cold boot. 100%

Update extern SD port setting from mmc2 to mmc0 to align with
before android release.

Signed-off-by: Ke Qinghua <qinghua.ke@freescale.com>
---
 evk_6sl/fstab.freescale | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/evk_6sl/fstab.freescale b/evk_6sl/fstab.freescale
index 77526e3..8e1a3c0 100644
--- a/evk_6sl/fstab.freescale
+++ b/evk_6sl/fstab.freescale
@@ -3,7 +3,7 @@
 # The filesystem that contains the filesystem checker binary (typically /system) cannot
 # specify MF_CHECK, and must come before any filesystems that do specify MF_CHECK
 
-/devices/soc0/soc.2/2100000.aips-bus/2198000.usdhc/mmc_host/mmc2 /mnt/media_rw/extsd vfat defaults voldmanaged=extsd:auto
+/devices/soc0/soc.2/2100000.aips-bus/2198000.usdhc/mmc_host/mmc1 /mnt/media_rw/extsd vfat defaults voldmanaged=extsd:auto
 /devices/soc0/soc.2/2100000.aips-bus/2184200.usb/ci_hdrc.1/usb1 /mnt/media_rw/udisk vfat defaults voldmanaged=udisk:auto
 /dev/block/mmcblk0p5    /system      ext4    ro                                                                               wait
 /dev/block/mmcblk0p4    /data        ext4    nosuid,nodev,nodiratime,noatime,nomblk_io_submit,noauto_da_alloc,errors=panic    wait,encryptable=footer
-- 
1.8.0

